### PR TITLE
[Nfs-Ganesha][Failover][Automation]Add new sceanrio

### PR DESCRIPTION
**Pull request by MohitBis**
[Nfs-Ganesha][Failover][Automation]Add new sceanrio
Scenarios:
1. Nfs-Ganesha perform failover when linux untar is running
2. Nfs-Ganesha perform failover and delete the export and when failback happens check the export list

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-IHWP84/Nfs-Ganesha_perform_failover_when_linux_untar_is_running_0.log
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-6DBCV5/Nfs-Ganesha_perform_failover_and_delete_the_export,_check_the_export_list_0.log

**Comment by MohitBis** [comment]
Updated Logs:

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-33CDL0/Nfs-Ganesha_perform_failover_when_linux_untar_is_running_0.log

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-6DBCV5/Nfs-Ganesha_perform_failover_and_delete_the_export,_check_the_export_list_0.log

**Review by pranavprakash20 — DISMISSED** [review]
LGTM

File: tests/nfs/nfs_ha_failover_delete_export.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            target=delete_export,
+            args=(
+                clients,
+                nfs_name,
+                nfs_export,
+            ),
+        )
+        th.start()
+
+        # Perfrom and verify failover
```
Comment (msainiRedhat) [inline tests/nfs/nfs_ha_failover_delete_export.py:82]: Deleting 1 export will happen quickly. Recommeneded way to test this scenario will be to create multiple exports and then perform failover. Deleting the export should run until the node is back online. Later check no stale export entry is present.

**Review by msainiRedhat — CHANGES_REQUESTED** [review]

File: tests/nfs/nfs_ha_failover_delete_export.py (hunk 1 above)
Comment (MohitBis) [reply tests/nfs/nfs_ha_failover_delete_export.py:81]: Updated.

Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-WACQRG/Nfs-Ganesha_perform_failover_and_delete_the_export,_check_the_export_list_0.log

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**